src/pybind/mgr/cephadm/agent.py

```diff
@@ -130,22 +146,112 @@ def shutdown(self) -> None:
         self.cherrypy_shutdown_event.set()
 
 
-class Root:
-    exposed = True
+class Root(object):
+
+    # collapse everything to '/'
+    def _cp_dispatch(self, vpath: str) -> 'Root':
+        cherrypy.request.path = ''
+        return self
 
     def __init__(self, mgr: "CephadmOrchestrator"):
         self.mgr = mgr
-        self.data = HostData(self.mgr)
 
-    def GET(self) -> str:
+    @cherrypy.expose
+    def index(self) -> str:
```

Comment:
Have you considered moving this up to the actual '/', and then the response could show the prometheus endpoints and the /data? I mean you have to know that the /prometheus endpoint exists first to get the help. Even if you don't include the /data endpoint having the breakdown of user accessible endpoints at '/' may make more sense

Reply:
Done. I've moved the `index` to  `/` but in the help I'm not showing `data` since this is an internal endpoint used by cephadm so I'd prefer to keep it hidden. The new layout is the following:

```
/  --> main/help page
/prometheus/sd-config?serivece=XXX
/prometheus/rules
/data --> not shown in the help page
```

Reply:
Thanks